packages/grid/src/vaadin-grid-styles.js

```diff
@@ -184,6 +184,10 @@ registerStyles(
       overflow: hidden;
     }
 
+    [part~='cell'][first-frozen-to-end] {
```

Comment:
Inside the grid's own styles, `[first-frozen-to-end]` should be specific enough as the selector.

Please move this next to the other related (frozen) styles.